devices/sprut.js

```diff
@@ -0,0 +1,45 @@
+const exposes = require('../lib/exposes');
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const reporting = require('../lib/reporting');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['WBMSW3'],
+        model: 'WBMSW3',
+        vendor: 'Sprut.device',
+        description: 'WB-MSW v.3 Zigbee Sensor',
+        fromZigbee: [fz.temperature, fz.illuminance, fz.humidity, fz.occupancy, fz.sprut_occupancy, fz.co2, fz.sprut_voc,
+            fz.sprut_noise, fz.sprut_noise_detected, fz.on_off],
+        toZigbee: [tz.on_off],
+        exposes: [e.temperature().withEndpoint(1), e.illuminance(), e.illuminance_lux(), e.humidity(),
```

Comment:
Please use only endpoint names which are defined in https://github.com/Koenkk/zigbee2mqtt/blob/42569d6c39295b6bcbde788a2db94e0262b434e8/lib/util/utils.ts#L30 (e.g. `l1` `l2`) and define them in `endpoint()` (e.g. https://github.com/Koenkk/zigbee-herdsman-converters/blob/2050b8fd7d0eaa48769f77bc0dbe89eb13e3bd2c/devices/tuya.js#L1290)

Reply:
Added endpoint names and removed unused clusters for now. Got an error:
```
Cannot read property 'bind' of undefined
```
Please help me.

[database.txt](https://github.com/Koenkk/zigbee-herdsman-converters/files/7737185/database.txt)
[zigbee2mqtt.log](https://github.com/Koenkk/zigbee-herdsman-converters/files/7737186/zigbee2mqtt.log)

Reply:
@Koenkk 
It turns out that getEndpoint() doesn't like it when they do this:
```
device.getEndpoint('default')
```
It is necessary to indicate the end point number:
```
device.getEndpoint(1)
```
It remains to solve something with an error when reconnecting the device:
```
Zigbee2MQTT:error 2021-12-17 19:23:23: Failed to read state of '0x842e14fffe8b184e' after reconnect (Read 0x842e14fffe8b184e/1 genOnOff(["onOff"], {"sendWhenActive":false,"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNSUPPORTED_ATTRIBUTE'))
```
[reconnect.log](https://github.com/Koenkk/zigbee-herdsman-converters/files/7737354/reconnect.log)